WTWSMS/decisions/realm_decisions.txt

```diff
@@ -662,7 +662,14 @@ decisions = {
 			has_global_flag = viking_age_started
 			NOT = { has_global_flag = varangian_guard_founded }
 			any_playable_ruler = {
-				religion = norse_pagan
+				OR = {
+					religion = norse_pagan
+					religion = germanic_pagan
+					religion = norse_pagan_reformed
+					religion = germanic_pagan_reformed
+					religion = saxon_pagan
+					religion = saxon_pagan_reformed
+				}
```

Comment:
I have done this many times: all things that apply to Norse Pagans should apply to Germanic Pagans, and vice-versa. Leave it as it is, please.

Actually, Saxon Paganism DOES exist in the religious code (under "west_pagan"), as it always has, and it represents Maelgwyn's Wodenic Paganism. I simply added it pending Saxon Paganism's implementation down the line. It doesn't hurt anything or change anything at this time, so why remove it?